src/bobbit/bot.py

```diff
@@ -109,6 +114,14 @@ def update_user_seen(self, user, timestamp):
         else:
             self.users[user]['last_seen'] = timestamp
 
+    async def send_mail(self, message, user):
```

Comment:
I don't think we should modify the internal bot class.  Instead keep the mail part contained in the module.

What you can do is have two hooks: 1. the current one to catch the "mail" command.  2. A general hook that checks the user who sent a message and sees if they have mail (look at the "lookup" module for how to catch any message).

Likewise, to keep things consistent with gonzobot, let's call this "tell" rather than "mail".